test/Graphics/Transform.cpp

```diff
@@ -0,0 +1,353 @@
+    }
+
+    SUBCASE("combine()")
+    {
+        auto identity = sf::Transform::Identity;
+        CHECK(identity.combine(sf::Transform::Identity) == sf::Transform::Identity);
+        CHECK(identity.combine(sf::Transform::Identity).combine(sf::Transform::Identity) == sf::Transform::Identity);
+
+        sf::Transform transform(1.0f, 2.0f, 3.0f,
+                                4.0f, 5.0f, 4.0f,
+                                3.0f, 2.0f, 1.0f);
+        CHECK(identity.combine(transform) == transform);
+        CHECK(transform.combine(sf::Transform::Identity) == transform);
+        CHECK(transform.combine(transform) == sf::Transform(18.0f, 18.0f, 14.0f,
+                                                            36.0f, 41.0f, 36.0f,
+                                                            14.0f, 18.0f, 18.0f));
+        CHECK(transform.combine(sf::Transform(10.0f,  2.0f,  3.0f,
+                                               4.0f, 50.0f, 40.0f,
+                                              30.0f, 20.0f, 10.0f))
+            == sf::Transform( 672.0f, 1216.0f,  914.0f,
+                             1604.0f, 2842.0f, 2108.0f,
+                              752.0f, 1288.0f,  942.0f));
+    }
+
+    SUBCASE("translate()")
+    {
+        sf::Transform transform(9, 8, 7, 6, 5, 4, 3, 2, 1);
+        CHECK(transform.translate({10.0f, 20.0f}) == sf::Transform(9, 8, 257, 6, 5, 164, 3, 2, 71));
+        CHECK(transform.translate({10.0f, 20.0f}) == sf::Transform(9, 8, 507, 6, 5, 324, 3, 2, 141));
+    }
+
+    SUBCASE("rotate()")
+    {
+        SUBCASE("About origin")
+        {
+            sf::Transform transform;
+            transform.rotate(90);
+            CHECK(transform.getMatrix()[0] == Approx(0));
+            CHECK(transform.getMatrix()[4] == Approx(-1));
+            CHECK(transform.getMatrix()[12] == Approx(0));
+            CHECK(transform.getMatrix()[1] == Approx(1));
+            CHECK(transform.getMatrix()[5] == Approx(0));
+            CHECK(transform.getMatrix()[13] == Approx(0));
+            CHECK(transform.getMatrix()[3] == Approx(0));
+            CHECK(transform.getMatrix()[7] == Approx(0));
+            CHECK(transform.getMatrix()[15] == Approx(1));
+        }
+
+        SUBCASE("About custom point")
+        {
+            sf::Transform transform;
+            transform.rotate(90, {1.0f, 0.0f});
+            CHECK(transform.getMatrix()[0] == Approx(0));
+            CHECK(transform.getMatrix()[4] == Approx(-1));
+            CHECK(transform.getMatrix()[12] == Approx(1));
+            CHECK(transform.getMatrix()[1] == Approx(1));
+            CHECK(transform.getMatrix()[5] == Approx(0));
+            CHECK(transform.getMatrix()[13] == Approx(-1));
+            CHECK(transform.getMatrix()[3] == Approx(0));
+            CHECK(transform.getMatrix()[7] == Approx(0));
+            CHECK(transform.getMatrix()[15] == Approx(1));
+        }
+    }
+
+    SUBCASE("scale()")
+    {
+        SUBCASE("About origin")
```

Comment:
I'm not sure what a good preposition is here, but I don't think it's "about".
Maybe "from" or "based in".

Reply:
"About" was something I often said in my engineering undergraduate courses in this context. Not sure if that's just an American English thing or something even more specific to the region where I went to school but as far as I'm aware it's a common way to refer to operation or transformation that take place with respect to a given point in space. For now I changed it to "Around".

Reply:
I see, thanks for the explanation! I never heard this term, but I'm also not a native English speaker. I'm fine with both "around" or "about" in that case.